exporters/logging/src/main/java/io/opentelemetry/exporters/logging/LoggingMetricExporter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.exporters.logging;
+
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import java.util.Collection;
+import java.util.logging.Logger;
+
+public class LoggingMetricExporter implements MetricExporter {
+  private static final Logger logger = Logger.getLogger(LoggingMetricExporter.class.getName());
+
+  @Override
+  public ResultCode export(Collection<MetricData> metrics) {
+    logger.info("Received a collection of " + metrics.size() + " metrics for export.");
+    for (MetricData metricData : metrics) {
+      logger.info("metric: " + metricData);
+    }
+    return ResultCode.SUCCESS;
+  }
+}
```

Comment:
I think we should plugin this with the interval reader so that it prints periodically to the logs.

Reply:
I think so too. Wasn't sure if it should be part of this PR, or a follow-on.

Reply:
also, that hasn't been merged yet. ;)

Reply:
Hey, why don't you use the `{}` placeholder? just to avoid the toString in case of warn log level

Reply:
Probably not a big deal with this particular use-case, since these exporters aren't really meant to be used outside of experimental/demo cases, and not in production. I can put a PR in to change them, though, as it is a better use-case.